webpack/base.config.js

```diff
@@ -44,7 +48,10 @@ module.exports = {
             loader: 'postcss-loader',
             options: {
               postcssOptions: {
-                plugins: ['postcss-preset-env', 'autoprefixer', 'pixrem'],
```

Comment:
Are we no longer using the removed plugins?

Reply:
Both `pixrem` and `postcss-preset-env` are relevant when we had raw css. With tailwindcss there should be no need for these. All the UI pages that got built by yume-labs used this config and no issue tied to this was observed. So I think it's a safe change